app.rb

```diff
@@ -0,0 +1,61 @@
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'rental'
+require_relative 'book'
+require_relative 'person'
+
+class App
+  def initialize
+    @books = []
+    @peoples = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: #{book.title} Author: #{book.author} \n" }
+  end
+
+  def list_peoples
+    @peoples.each do |people|
+      puts "[#{people.class.name}] Name: #{people.name} ID: #{people.id} Age: #{people.age} \n"
+    end
+  end
+
+  def create_teacher(age, specialization, parent_permission, name)
+    @peoples << Teacher.new(age, specialization, parent_permission: parent_permission, name: name)
+  end
+
+  def create_student(age, name, parent_permission)
+    @peoples << Student.new(age, name: name, parent_permission: parent_permission)
+  end
+
+  def create_book
+    puts 'Title:'
+    title = gets.chomp
+    puts 'Author:'
+    author = gets.chomp
+    @books << Book.new(title, author)
+  end
+
+  def create_rental
+    puts 'Select a book from the following list by number'
+    @books.each.with_index { |book, idx| puts "#{idx}) Book #{book.title} by #{book.author}" }
+    book_index = gets.chomp.to_i
+    puts 'Select a person from the following list by number(not ID)'
+    @peoples.each.with_index do |person, idx|
+      puts "#{idx}) [#{person.class.name}] Name #{person.name}, ID #{person.id}, Age #{person.age}"
+    end
+    person_index = gets.chomp.to_i
+    puts 'Date:-'
+    date = gets.chomp
+    Rental.new(@books[book_index], @peoples[person_index], date)
+  end
```

Comment:
Your app console
![image](https://github.com/Mikey1-nuel/school_library/assets/78251927/2dc5e605-f339-4ddf-8033-aee3f00657aa)

Currently, the method `def create_rental` is not working, and creating a rental crash in the app, please kindly fix this issue to ensure that the console app works as expected.. 💪☺